src/master/connecting/sdks.md

```diff
@@ -19,6 +19,7 @@ The most well-known and recommended immudb SDK is written in [Golang](https://go
 | `go`               | immudb team  | 1.3.2       |     [link](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client)  |                                   |
 | `python`               | immudb team  | 1.3.2       |  [link](https://github.com/codenotary/immudb-py) |                                     |
 | `JAVA`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
+| `.NET`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4net)  |                                      |
```

Comment:
```suggestion
| `.NET`               | immudb team  | 1.4.0       |   [link](https://github.com/codenotary/immudb4net)  |                                      |
```

(and I should have updated the go version :facepalm:)

Reply:
done